components/function-controller/design/scaling_functions.md

```diff
@@ -0,0 +1,40 @@
+# Serverless Functions Scaling Modes
+
+## Summary
+Initially, the Serverless API supported the `spec.ScaleConfig` field only. The work flow was as follow:
```

Comment:
I changed lines 4-8 to past tense, because we're talking about the previous workflow, correct?